src/Services/Air/templates/AIR_LOW_FARE_SEARCH_ASYNCH_REQUEST.handlebars.js

```diff
@@ -0,0 +1,105 @@
+module.exports = `
```

Comment:
Please fix `asynch` in file name

Reply:
Does this template have a lot of changes comparing to regular LFS?
Is there any need to keep two version of them? IMO it's nice to have one and include some blocks based on request type.

Reply:
Commit 1a5ffed merged async template to lfs template, no more file named asynch.